docs/content/components/dialog.md

```diff
@@ -468,6 +468,26 @@ const container = ref(null)
 
 Adheres to the [Dialog WAI-ARIA design pattern](https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal).
 
+### Close icon button
+
+When providing an icon (or font icon), remember to label it correctly for screen reader users.
+
+```html line=8-9
+<DialogRoot>
+  <DialogTrigger />
+  <DialogPortal>
+    <DialogOverlay />
+    <DialogContent>
+      <DialogTitle />
+      <DialogDescription />
+      <DialogClose aria-label="Close">
+        <span aria-hidden>×</span>
+      </DialogClose>
+    </DialogContent>
+  </DialogPortal>
+</DialogRoot>
+```
+
```

Comment:
I added a note in the docs to remind authors to give their 'Close' icon button an accessible name as well (copied and pasted from Toast)! Let me know if it should be in a separate PR.

I'd do it for AlertDialog and Popover too, but I'm not sure if that'd be too out of scope. (Anyway, I could not get `pnpm docs:dev` to work 🙈 But it's late and I'll try again another time.)

Reply:
Nic addition! Let's handle AlertDialog and Popover in another PR ya 😁
Have you tried `pnpm i` then `pnpm docs:dev`?

Reply:
Yep, I ran `pnpm i` and then `pnpm docs:dev`, both in the root folder and the docs folder. The error/warning was that the entrypoint of 'radix-vue' couldn't be resolved, or something along those lines.

I will say that I've never used pnpm before, I installed it yesterday for the first time ever 🙈 Maybe I should have tried restarting my machine at some point. Anyway, not to worry now, I will ask for help in the Discord server if/when I do try again!